kms-message/src/kms_kmip_item_type_private.h

```diff
@@ -17,17 +17,43 @@
 #ifndef KMS_KMIP_ITEM_TYPE_PRIVATE_H
 #define KMS_KMIP_ITEM_TYPE_PRIVATE_H
 
-typedef enum {
-   KMIP_ITEM_TYPE_Structure = 0x01,
-   KMIP_ITEM_TYPE_Integer = 0x02,
-   KMIP_ITEM_TYPE_LongInteger = 0x03,
-   KMIP_ITEM_TYPE_BigInteger = 0x04,
-   KMIP_ITEM_TYPE_Enumeration = 0x05,
-   KMIP_ITEM_TYPE_Boolean = 0x06,
-   KMIP_ITEM_TYPE_TextString = 0x07,
-   KMIP_ITEM_TYPE_ByteString = 0x08,
-   KMIP_ITEM_TYPE_DateTime = 0x09,
-   KMIP_ITEM_TYPE_Interval = 0x0A
-} kmip_item_type_t;
+#include "kms_message/kms_message_defines.h"
+
+#define KMS_XMACRO           \
```

Comment:
Should this be preceeded by a check/assertion that `KMS_XMACRO` has not already been defined?